src/frontend/utils/umami/useUmami.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+
+import { erProd } from '../miljø';
+
+export const useUmami = () => {
+    useEffect(() => {
+        const umamiId = erProd()
+            ? '4faef03-4e3b-4783-b1de-1df31f8695a1'
+            : '4700b5e8-da50-4dfe-9884-219c02374cb0';
```

Comment:
Er dette IDn som er unike for våret prosjekt?

Reply:
Jeps, for prod og dev